src/errors/Messages.js

```diff
@@ -17,6 +17,8 @@ const Messages = {
 
   SHARDING_INVALID: 'Invalid shard settings were provided.',
   SHARDING_REQUIRED: 'This session would have handled too many guilds - Sharding is required.',
+  INVALID_INTENTS: 'There is an invalid intent for websocket intents.',
```

Comment:
"You've provided an invalid intent to the WebSocket."

Reply:
```suggestion
  INVALID_INTENTS: 'Invalid intent provided for WebSocket intents.',
```